### DISCUSSION TITLE
Добавление нового ремесла

### DISCUSSION THREAD

**Discussion by PingwinGrisha** [octothorp-team/public-dobrograd#661] [Предложения]
Очень не хватает разнообразия в алкоголе, подумал, что было бы неплохо добавить хотя бы несколько новых предметов (например Виски, Абсент и т.п.). В идеале еще добавить систему изготовления алкоголя: брожение, перегонка и т.п., чтобы производить собственный алкоголь. Можно было бы варить пиво, гнать самогон, делать вино и виски в бочках, но это прям в идеале, для начала просто добавить возможность купить больше видов алкоголя в магазине - отличный шаг

**Comment by tapo4ekdinos** [comment]
Не помешало бы еще увеличить длительность опьянения, сейчас она маленькая
